extensions/KHR/EGL_KHR_platform_wayland.txt

```diff
@@ -93,6 +94,16 @@ New Behavior
     until the next buffer swap. The rationale behind this behavior is to keep
     operations result accurate until the next swap.
 
+    During the call to eglSwapBuffers wl_surface.attach will be requested with
```

Comment:
Should we maybe clarify that the EGL implementation is the one who will do these requests instead of just saying they "will be requested"? It seems like that's the way things like this are worded in some other extensions for clarity. 
```suggestion
    During the call to eglSwapBuffers wl_surface.attach will be requested by the EGL implementation with
```
Not sure if we also need to call out things like explicit sync points being sent by the implementation as part of this commit? I guess what you have right now just specifies the minimum set of requests the implementation will do, so that could be fine.

Reply:
It seems hard both to enumerate all the state ever needed and to write general language that implies the right amount of state.

Maybe for explicit sync:

"if the EGL implementation associated a wp_linux_drm_syncobj_surface_v1 object with the surface, an acquire and a release point will be associated with the surface prior to the commit as well."

For others state the implementation might need to set as part of a commit, hopefully it makes more sense to call it out in separate extensions? E.g., as an interaction with this extension in a present timing extension, an HDR extension, etc., or in the other direction for existing EGL extensions that necessitate additional wayland surface state?

Reply:
I reworded things a bit while still trying to make it more explicit, because adding "by the EGL implementation" to every "requested" would have been a bit much.

Reply:
Didn't add anything about explicit sync there, as it's not strictly needed, but could be added too I guess.

Reply:
> "if the EGL implementation associated a wp_linux_drm_syncobj_surface_v1 object with the surface, an acquire and a release point will be associated with the surface prior to the commit as well."

Added this as well, thanks.